GitUI/CommandsDialogs/SettingsDialog/Pages/ChecklistSettingsPage.cs

```diff
@@ -447,7 +447,8 @@ private bool CheckSSHSettings()
             }
 
             var ssh = _sshPathLocator.Find(AppSettings.GitBinDir);
-            RenderSettingSet(SshConfig, SshConfig_Fix, string.IsNullOrEmpty(ssh) ? _opensshUsed.Text : string.Format(_unknownSshClient.Text, ssh));
+            var sshPath = _sshPathLocator.Find(AppSettings.GitBinDir);
```

Comment:
`_sshPathLocator.Find` return a valid path or an empty string, if SSH is not found. Why do we need to detect it the second time?

Reply:
To find if this is OpenSSH or not.
This will be changed (see above), probably returning empty for OpenSSH.